lib/Core/SQLiteBuildDB.cpp

```diff
@@ -495,24 +526,44 @@ class SQLiteBuildDB : public BuildDB {
     assert(result == SQLITE_OK);
     result = sqlite3_step(insertIntoKeysStmt);
     if (result != SQLITE_DONE) {
-      *error_out = getCurrentErrorMessage();
-      return UINT64_MAX;
+      // FIXME: We need to support error reporting here, but the engine cannot
```

Comment:
Will do... I didn't want to put it back, but the only other option was to add an abort at the level above it so it didn't seem worth it. rdar://problem/32166414